CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 28.0.6
+
+### Fixed
+
+- Dependency updates
```

Comment:
Found it below the file, maybe let's keep the syntax?
```suggestion
### Changed

- Upgrade dependencies
```

Reply:
yes... both are in there... one way for 27.0.0 - the other for 27.0.1 - I don't have a preference myself.